form/3d/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: Lines for Space
+tags: [P5.js, Form, Law of Space]
+permalink: https://github.com/fabianmoronzirfas/gestalten-in-code/tree/master/form
```

Comment:
The permalink should be the category and your title e.g. 

    form/lines-for-space/